GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -40,16 +40,35 @@ public enum ToolBarSelectedButton
         private const int ButtonWidth = 200;
         private const int LongButtonWidth = 300;
         private const int ShortButtonWidth = 100;
-        
+
         private const string LoadingScreenErrorTitle = "Creating Loading Scene Error";
         private const string OkButtonText = "OK";
 
```

Comment:
Add a comment:

// Local copy of the fields from QuickDeployConfig.Config to track unsaved changes.